docs/deploying.md

```diff
@@ -0,0 +1,98 @@
+# Deploying
+
+When it comes time to share your project, you have a few options for deploying it for others to see. Framework is compatible with many static site hosts and automation environments, but here we'll focus on using GitHub Actions to deploy to Observable.
+
+## Manually deploying to Observable
+
+If you don't already have a project to deploy, you can make one by following [getting-started](./getting-started). First, make sure that your project builds without error:
+
+```sh
+$ npm run build
+```
+
+Once that is done you can deploy to Observable with the command
+
+```sh
+$ npm run deploy
+```
+
+The first time you run this it will prompt you for details needed to set up the project on the server such as the project's _slug_ (which determines its URL) and the access level. If you don't already have an Observable account or aren't signed in, this command will also guide you through setting that up.
```

Comment:
```suggestion
The first time you run this, it will prompt you for details needed to set up the project on the server, such as the project's _slug_ (which determines its URL), and the access level. If you don’t already have an Observable account or aren’t signed in, this command will also guide you through setting that up.
```